README.md

```diff
@@ -161,6 +161,34 @@ Parameters should be defined as follows:
   Note that after test case ID asterisk should be added. This is necessary due
   to the construction of the flag `-t` (or `--test`)
 
+#### Known issues
+
+When trying to run tests, you might face errors related to libraries not
+linking properly. You can try to fix that issue by doing:
+
+```bash
+source ${PWD}/venv/bin/activate
```

Comment:
This looks quite odd and I would not recommend this

Reply:
removed [here](https://github.com/Dasharo/open-source-firmware-validation/pull/12/commits/263532fb3c2da8abbd4c47df686e810a2fb0920f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L170)